recipes/discord/webview.js

```diff
@@ -33,9 +33,14 @@ module.exports = (Ferdium, settings) => {
 
     if (link || button) {
       const url = link ? link.getAttribute('href') : button.getAttribute('title');
-      const stayInsideDiscord = url.includes('https://discordapp.com/channels/');
+      const skipDomains = [/^https:\/\/discordapp\.com\/channels\//i, /^https:\/\/discord\.com\/channels\//i];
 
-      if (!Ferdium.isImage(url) && !stayInsideDiscord) {
+      skipDomains.every(skipDomain => {
```

Comment:
i think you want to break out of the loop as soon as any of the discord domains are hit. The current loop will take the last value for `stayInsideDiscord`

Reply:
@SpecialAro - pls see this comment

Reply:
I've tested it and, as is, it will break as soon as `!stayInsideDiscord` is false, so there's no issue (I was surprised as well with this ahah)

Reply:
> I've tested it and, as is, it will break as soon as `!stayInsideDiscord` is false, so there's no issue (I was surprised as well with this ahah)

Yep, I did my research @vraravam. I picked every() specifically because the second it gets a false return it breaks the cycle so this way we could do it in fewer steps and less chances of bugs. 😁